tools/doc-pol_new.yaml

```
@@ -0,0 +1,2296 @@
+convert_library_version: 'v2 ; Compat Mode: [0] {[v2.1] (DEFAULT) Don''t use any Compatibility
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)